src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -36,6 +37,8 @@ internal partial class CurlHandler : HttpMessageHandler
         private const string NoContentType = HttpKnownHeaderNames.ContentType + ":";
         private const int CurlAge = 5;
         private const int MinCurlAge = 3;
+        private const int SecurePasswordByteLength = 39;
+        private const string CertificateFileType = "P12";
```

Comment:
If we're only supporting OpenSSL as the backend, and they have the callbacks, why do we still want/need the PFX exporting code?